applications/crossbar/src/modules/cb_cdrs.erl

```diff
@@ -520,12 +520,26 @@ col_account_call_type(JObj, _Timestamp) -> wh_json:get_value([<<"custom_channel_
 col_rate(JObj, _Timestamp) -> wh_util:to_binary(wht_util:units_to_dollars(wh_json:get_value([<<"custom_channel_vars">>, <<"rate">>], JObj, 0))).
 col_rate_name(JObj, _Timestamp) -> wh_json:get_value([<<"custom_channel_vars">>, <<"rate_name">>], JObj, <<>>).
 col_bridge_id(JObj, _Timestamp) -> wh_json:get_value([<<"custom_channel_vars">>, <<"bridge_id">>], JObj, <<>>).
-col_recording_url(JObj, _Timestamp) -> wh_json:get_value([<<"recording_url">>], JObj, <<>>).
+col_recording_url(JObj, _Timestamp) -> gen_recording_url(wh_json:get_value([<<"custom_channel_vars">>, <<"media_transfer_destination">>], JObj, <<>>), wh_json:get_value([<<"custom_channel_vars">>, <<"media_name">>], JObj, <<>>)).
 col_call_priority(JObj, _Timestamp) -> wh_json:get_value([<<"custom_channel_vars">>, <<"call_priority">>], JObj, <<>>).
 
 col_reseller_cost(JObj, _Timestamp) -> wh_util:to_binary(reseller_cost(JObj)).
 col_reseller_call_type(JObj, _Timestamp) -> wh_json:get_value([<<"custom_channel_vars">>, <<"reseller_billing">>], JObj, <<>>).
 
+%%%
+%%% generates a recording_url from the media_transfer_destination and media_file
+%%%
+-spec gen_recording_url(ne_binary(), ne_binary()) -> ne_binary().
```

Comment:
Spec is wrong: replace the 3 `ne_binary()` with `binary()` (but I'd use `api_binary()` and return `'undefined'` instead of `<<>>`).